validator/api/src/main/java/tech/pegasys/teku/validator/api/ValidatorConfig.java

```diff
@@ -421,8 +422,13 @@ public Builder proposerDefaultFeeRecipient(final String proposerDefaultFeeRecipi
       if (proposerDefaultFeeRecipient == null) {
         this.proposerDefaultFeeRecipient = Optional.empty();
       } else {
-        this.proposerDefaultFeeRecipient =
-            Optional.of(Eth1Address.fromHexString(proposerDefaultFeeRecipient));
+        try {
+          this.proposerDefaultFeeRecipient =
+              Optional.of(Eth1Address.fromHexString(proposerDefaultFeeRecipient));
```

Comment:
I'd be very tempted to just make this method take an `Eth1Address` instead of a `String` and make the CLI type an `Eth1Address` but we're getting well outside the scope of this PR now.